.azure-pipelines/build.yml

```diff
@@ -47,6 +47,27 @@ jobs:
   - checkout: self
     clean: true
     submodules: true
+  - task: DownloadPipelineArtifact@2
+    inputs:
+      source: specific
+      project: build
+      pipeline: Azure.sonic-buildimage.common_libs
+      runVersion: 'latestFromBranch'
+      runBranch: 'refs/heads/master'
+      path: $(Build.ArtifactStagingDirectory)/download
+      ${{ if eq(parameters.arch, 'amd64') }}:
+        artifact: common-lib
+      ${{ else }}:
+        artifact: common-lib.${{ parameters.arch }}
+      patterns: |
+        target/debs/buster/libyang-*.deb
+        target/debs/buster/libyang_*.deb
```

Comment:
This should be `bullseye`, not `buster`.

Reply:
The reason we using buster is because today the swss-common lib download by next step is build with buster.